app/presenters/content_item/contents_list.rb

```diff
@@ -47,14 +47,19 @@ def first_item_has_long_content?
     def first_item_content
       element = first_item
       first_item_text = ""
-      allowed_elements = %w[p ul ol]
-
-      until element.name == "h2"
-        first_item_text += element.text if element.name.in?(allowed_elements)
+      allowed_elements = %w[p ul ol div]
```

Comment:
`div` is checked separately in line 53, so adding it here doesn't change anything and is confusing, so it can be removed from this line. So link 55 can be also changed to not exclude it.